src/config/config.php

```diff
@@ -11,9 +11,12 @@
      *     'forum/thread',
      * ],
      */
-
     'messages' => [
 
     ],
 
+    /**
```

Comment:
Config files are using single /* for comments. Look on Laravel config files for example.